api/src/test/java/com/epam/pipeline/manager/cluster/writer/MonitoringStatsWriterTest.java

```diff
@@ -49,7 +49,7 @@
     @Test
```

Comment:
Probably we can rename this test class to `CsvMonitoringStatsWriterTest` as long as it tests only the csv functionality.

Reply:
Done